lms/djangoapps/commerce/utils.py

```diff
@@ -76,10 +77,16 @@ def payment_page_url(self):
         """
         return self.get_absolute_ecommerce_url(self.config.single_course_checkout_page)
 
-    def checkout_page_url(self, sku):
-        """ Construct the URL to the ecommerce checkout page and include a product.
+    def checkout_page_url(self, *args):
```

Comment:
I would rename `args` to `skus` here seeing that only SKUs should be sent to this method. Also add it to the docstring.